### PR TITLE
community: List our Named supporters and link to GH Sponsors

### PR DESCRIPTION
**Pull request by liskin**
We promised to list Named sponsors as supporters, so we better deliver soon. :-)

Rendered here: https://liskin.github.io/xmonad-web/community.html

Suggestions to improve the wording very welcome, I'm extremely bad at non-technical writing.

---

#### [community: List our Named supporters and link to GH Sponsors](../commit/4f8aa6476e06a67db53d8d4f87d3fcaeea7aa6fd)

This comes with code to automatically regenerate the list, but GitHub
currently exposes private sponsors without any way to distinguish them
from public, so I'm not including any GH Actions workflow to regenerate
this periodically, as this still unfortunately needs human oversight.

### CODE REVIEW

**Review by TheMC47 — APPROVED** [review]
LGTM, thanks a lot

File: community.md [hunk 1]
```diff
@@ -8,71 +8,86 @@
...
+<ul>
+{% for sponsor in site.data.sponsors-named %}
+<li><a href="{{ sponsor.websiteUrl | default: sponsor.url }}">{{ sponsor.name | default: sponsor.login | escape }}</a></li>
+{% endfor %}
+<li>and many <a href="https://github.com/sponsors/xmonad#sponsors">others</a></li>
+</ul>
+
+We're grateful for this support, and we humbly ask you to consider [becoming a
+supporter as well](https://github.com/sponsors/xmonad) if you aren't already.
+Thank you!
```
Comment (slotThe) [inline community.md:90]: I think this reads just fine!  Off the cuff, an alternative suggestion would be something like

> We're incredibly grateful for this support. If you also want to aid our efforts going forward, consider [becoming a supporter](https://github.com/sponsors/xmonad) today[.!] Thank you!

But I'm not sure if this is really _better_, as opposed to just being _different_.

**Review by slotThe — APPROVED** [review]
LGTM as well; thanks!

File: community.md (hunk 1 above)
Comment (liskin) [reply community.md:90]: I can't really decide either, so I'm going to merge this as is, and think about it more later. I also just realized that we might want to mention Open Collective as well: it's possibly a better way for companies/orgs to contribute (smaller fee) and it shows our finances are transparent. And I'll play a bit more with GH tokens and the automatic refresh of the sponsors list (for which I need the default branch of my fork).

Comment (liskin) [reply community.md:90]: Ended up using your wording in the release announcement to spice things up a little. :-))

**Pull request merged**